tests/tasks/generators/manual-deployment/deploy-pods-with-size.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: deploy-pods-with-size
+  namespace: tekton-pipelines
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/categories: Kubernetes
+    tekton.dev/tags: CLI, kubectl
+    tekton.dev/displayName: "kubernetes actions"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: |
+    Deploy multiple pods with given size in the test cluster.
+  params:
+  - name: name
+    default: "guest"
+    description: Name of the guest cluster
+  - name: number-of-pods
+    default: "50"
+    description: Number of dummy pods
+  - name: pod-size
+    default: "5000"
+    description: Size of the pod object
```

Comment:
is this is MB ? or Bytes ? Helps if it's more descriptive ? wdyt ?

Reply:
Agree. Done.